# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1436374115 14400
#      Wed Jul 08 12:48:35 2015 -0400
# Node ID 820c5a4dd0abc5f2d1704c94e76dbb53993a1094
# Parent  03c88436ad8c32051087b1b4515574ff28927712
use $(abs_top_srcdir) in TEXINPUTS for processing refcard sources

* doc/module.mk ($(refcard_DVI), $(refcard_PDF)): use
$(abs_top_srcdir) instead of $(srcdir) to set TEXINPUTS.

diff --git a/doc/module.mk b/doc/module.mk
--- a/doc/module.mk
+++ b/doc/module.mk
@@ -460,22 +460,22 @@ clean-local: interpreter-clean
 
 maintainer-clean-local: doxyhtml-maintainer-clean
 
 $(srcdir)/doc/interpreter/images.mk: $(srcdir)/doc/interpreter/config-images.sh $(srcdir)/doc/interpreter/images.awk $(srcdir)/doc/interpreter/images
 	$(srcdir)/doc/interpreter/config-images.sh $(top_srcdir)
 
 $(refcard_DVI) : %.dvi : %.tex doc/refcard/$(octave_dirstamp)
 	-$(AM_V_TEX)cd $(@D) && \
-	TEXINPUTS="$(srcdir)/doc/refcard:$(TEXINPUTS):" \
+	TEXINPUTS="$(abs_top_srcdir)/doc/refcard:$(TEXINPUTS):" \
 	$(TEX) $(<F) $(AM_V_texidevnull)
 
 $(refcard_PDF) : %.pdf : %.tex doc/refcard/$(octave_dirstamp)
 	-$(AM_V_PDFTEX)cd $(@D) && \
-	TEXINPUTS="$(srcdir)/doc/refcard:$(TEXINPUTS):" \
+	TEXINPUTS="$(abs_top_srcdir)/doc/refcard:$(TEXINPUTS):" \
 	$(PDFTEX) $(<F) $(AM_V_texidevnull)
 
 endif
 
 doc/interpreter/undocumented_list:
 	rm -f $@-t $@
 	-$(PERL) $(srcdir)/doccheck/mk_undocumented_list > $@-t
 	mv $@-t $@
